container/service.proto

```diff
@@ -272,12 +286,14 @@ message GetExtendedACLRequest {
 
 // Get Extended ACL
 message GetExtendedACLResponse {
-  // Response body
+  // Get Extended ACL Response body can be empty if the requested container did
+  // not have Extended ACL Table attached or Extended ACL was not allowed at
+  // container creation.
```

Comment:
> or Extended ACL was not allowed at container creation.

Is there any ways to do that, actually? Container contract does not checks container fields such as basic ACL to make such restriction. You can set extended ACL but it will have no power.

Reply:
Agree. I meant that it will be ignored if not allowed in `BasicACL`. Will fix.